### ISSUE TITLE
[BUG] - RemoveJob does not work in v0.5.1

### ISSUE THREAD

**Issue by huage1994** [go-co-op/gocron#107]
### Describe the bug

Method RemoveJobByTag does not work in v0.5.1 ,  it can work in v0.4.0

### To Reproduce

run code in v0.4.0

```
package main

import (
	"fmt"
	"github.com/go-co-op/gocron"
	"time"
)

func main() {
	s1 := gocron.NewScheduler(time.UTC)
	tag1 := []string{"tag1"}
	tag2 := []string{"tag2"}
	s1.Every(3).Seconds().SetTag(tag1).Do(func() {
		fmt.Println(len(s1.Jobs()), "I am job 1", time.Now())
	})

	jobz, _ := s1.Every(3).Seconds().SetTag(tag2).Do(func() {
		fmt.Println(len(s1.Jobs()), "I am job 2", time.Now())
	})
	print(jobz)
	s1.StartAsync()

	time.Sleep(3 * time.Second)

	s1.RemoveJobByTag("tag2")

	fmt.Println(s1.Jobs())
	print("------------------------")

	time.Sleep(100 * time.Second)

}
```

**Comment by Streppel** [comment]
@huage1994 thanks for reporting it! Will take a look.

**Comment by Nikkhah** [comment]
Hi
I have this problem too. Could you find the solution?

**Comment by huage1994** [comment]
> Hi
> I have this problem too. Could you find the solution?

Just now upload fixed code to my forked repo.

**Comment by Streppel** [comment]
Sorry guys I'm short on time, as soon as I get my eyes on this I'll let you know here. Probably later today

**Comment by JohnRoesler** [comment]
This is released in [v0.6.0](https://github.com/go-co-op/gocron/releases/tag/v0.6.0)!
